src/datadoc/backend/dapla_dataset_path_info.py

```diff
@@ -406,3 +412,28 @@ def dataset_version(
             ):
                 return last_filename_element[1:]
         return None
+
+    @property
+    def dataset_shortname(
+        self,
+    ) -> str | None:
+        """Extract version information if exists in filename.
```

Comment:
I think this is left over from a copy/paste?

Just update it to describe the information returned from this method. We could also mention that it is expected to be the part before the dataset_state based on Dapla's naming convention.

Reply:
Final commit adds the correct description 😊